javascript/javascript1/week2/main.js

```diff
@@ -0,0 +1,106 @@
+//freeCodeCamp user:@TatjanaOcedova
+
+//Flight booking fullname
+function getFullname(firstname, surname, useFormalName, gender) {
+  if (firstname.length === 0 || surname.length === 0) {
+    return "Fullname is empty";
+  } else if (useFormalName == true) {
+    switch (gender) {
+      case "female":
+        return "Lady " + firstname + " " + surname;
+        break;
+      case "male":
+        return "Lord " + firstname + " " + surname;
+        break;
+      default:
+        return firstname + " " + surname;
+        break;
+    }
+  } else {
+    return firstname + " " + surname;
+  }
+}
```

Comment:
This looks good.
There is no reason to use break, if you return right over it. This is code that is never reached.

Instead of using +" "+, you could look into Template literals.

Look's great :)